server/src/main/resources/org/cloudfoundry/identity/uaa/db/postgresql/V4_99_1575367461__revocable_token_index.sql

```diff
@@ -1 +1 @@
-CREATE INDEX CONCURRENTLY IF NOT EXISTS revocable_tokens_user_id_client_id_response_type_identity__idx on revocable_tokens(user_id, client_id, response_type, identity_zone_id);
+CREATE INDEX IF NOT EXISTS revocable_tokens_user_id_client_id_response_type_identity_idx ON revocable_tokens(user_id, client_id, response_type, identity_zone_id);
```

Comment:
This isn't safe from a "no downtime" perspective since the table will be locked. I believe this is why the migration was written this way to begin with, correct?

Is there no way to address the test without changing the implementation?

Reply:
then we would have to change/adapt the failing test in Mfa context.-